src/scene/graph-node.js

```diff
@@ -480,6 +480,37 @@ class GraphNode extends EventHandler {
         return this;
     }
 
+
+    /**
+     * Detach a GraphNode from the hierarchy. Then recursively destroy all ancestors.
```

Comment:
Shouldn't this say descendants instead of ancestors?

Reply:
Wow, totally, thank you for seeing that! It's the original comment from here: https://github.com/playcanvas/engine/blob/039da2e23ca4a6dcc71630e5f3e79afe21553001/src/framework/entity.js#L502-L503

Do you want to make a code suggestion? "recursively destroy all children"?